init-scripts/init_postgres_schema.sql

```diff
@@ -0,0 +1,16 @@
+-- Connect to ODS database
+\connect ods
+
+-- Create schema
+CREATE SCHEMA IF NOT EXISTS FOREST;
+
+-- Create user
+CREATE USER ods_proxy_user_lrm WITH PASSWORD 'ods_proxy_user_lrm';
+
+-- Grant privileges
+GRANT USAGE ON SCHEMA FOREST TO ods_proxy_user_lrm;
```

Comment:
Great set up for testing! Schemas in ODS follow naming convention <app>_replication so you will likely be dealing with fta_replication and lrm_replication :)

Reply:
Thanks @abimichel! I updated the schema name to `lrm_replication`